snow/engine/snowman/transitive_test.go

```diff
@@ -1190,9 +1190,6 @@ func TestEngineRetryFetch(t *testing.T) {
 			IDV:     ids.GenerateTestID(),
 			StatusV: choices.Unknown,
 		},
-		ParentV: gBlk.ID(),
-		HeightV: 1,
-		BytesV:  []byte{1},
```

Comment:
So I understand these fields are not really used, but I feel they provide context that make the test more readable? They seems to tell intentions of test creator. I'd like to keep them, but open to discussion